docs/cloud-native-security/get-started-with-kspm.asciidoc

```diff
@@ -0,0 +1,186 @@
+                "elasticloadbalancing:DescribeLoadBalancers",
+                "ecr-public:DescribeRepositories",
+                "eks:DescribeNodegroup",
+                "ecr:DescribeImages",
+                "elasticloadbalancing:DescribeLoadBalancerPolicies",
+                "ecr:DescribeRepositories",
+                "eks:DescribeCluster",
+                "eks:ListClusters",
+                "elasticloadbalancing:DescribeInstanceHealth",
+                "ecr:GetRepositoryPolicy"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+[discrete]
+[[kspm-use-keys-directly]]
+===== Option 1 - Use access keys directly:
+Access keys are long-term credentials for an IAM user or the AWS account root user. To use access keys as credentials, you need to provide:
+
+  * `Access key ID`: The first part of the access key.
+  * `Secret Access Key`: The second part of the access key.
+
+For more details refer to https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys[AWS Access Keys and Secret Access Keys].
+
+IMPORTANT: You must select "Programmatic access" when creating the IAM user.
+
+[discrete]
+[[kspm-use-temp-credentials]]
+===== Option 2 - Use temporary security credentials:
+Temporary security credentials can be configured in AWS to last for some period of time. They consist of an access key ID, a secret access key, and a security token, which is typically found using `GetSessionToken`.
+
+NOTE: IAM users with multi-factor authentication (MFA) enabled need to submit an MFA code when calling `GetSessionToken`. For more details refer to https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[AWS Temporary Security Credentials].
+
+You can use the AWS CLI to generate temporary credentials. For example, you could use the following command if you have MFA enabled: `sts get-session-token --serial-number arn:aws:iam::1234:mfa/your-email@example.com --duration-seconds 129600 --token-code 123456`
+
+The output from this command should include the following fields, which you should supply to the KSPM integration:
+
+`Access key ID`: The first part of the access key.
+`Secret Access Key`: The second part of the access key.
+`Session Token`: A token required when using temporary security credentials.
+
+Because temporary security credentials are short term, once they expire you will need to generate new ones and manually update the integration's configuration to continue collecting cloud posture data. Update the credentials before they expire to avoid data loss.
+
+[discrete]
+[[kspm-use-a-shared-credentials-file]]
+===== Option 3 - Use a shared credentials file:
+If you use different AWS credentials for different tools or applications, you can use profiles to define multiple access keys in the same configuration file. For more details refer to https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html#file-format-creds[Create Shared Credentials File]
```

Comment:
Linking to the top-level page (instead of a subheading) might help if the 3rd party page gets restructured or moved, and we should give some guidance to help the reader find the docs if the link breaks. Also make it clearer that this links to AWS docs.
```suggestion
If you use different AWS credentials for different tools or applications, you can use profiles to define multiple access keys in the same configuration file. For more details, refer to the AWS documentation on https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html[shared credentials files].
```
IMO we could probably omit the link and just describe the AWS docs that the user should consult, but there might be a stronger need for this that I'm not aware of.